conf/messages.en

```diff
@@ -119,9 +119,7 @@ index.heading = pension-scheme-return-frontend
 index.guidance = Welcome to your new frontend. Please see the README file for a guide to getting started.
 
 checkYourAnswers.title = Check your answers
-changeYourAnswers.title = Change your answers
```

Comment:
"SchemeMemberDetailsAnswersController" this is still in use by the controller, Still has the generic "Check your answers" Does this need to be removed or updated to reflect the similar changes made in the ticket?

Reply:
Hi @Arei1997 , "SchemeMemberDetailsAnswersController" we can leave this one as a generic "Check your answers", because in the file they didn't mentioned to change this Controller, so @Vamshi-Muniganti will check with design team when they come back from holidays.